amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -0,0 +1,157 @@
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the Other Connector Fields>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Using Anypoint Studio to Configure a Connector].
+If, after reading this topic, you need additional information about the connector fields, see the xref:amazon-s3-connector-reference.adoc[Amazon S3 Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Amazon S3 to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `amazon s3` in the search field.
+. Click *Amazon S3* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Amazon S3:
+
+* *On Deleted Object*
++
+Initiates access to your app when an Amazon S3 object is deleted
++
+* *On New Object*
++
+Initiates access to your app when an Amazon S3 object is created
+
+For example, to configure an *On Deleted Object* source, follow these steps:
+
+. In the *Mule Palette* view, select *Amazon S3 > On Deleted Object*.
+. Drag *On Deleted Object* to the Studio canvas.
+. On the *On Deleted Object* configuration screen, optionally change the value of the *Display Name* field.
+. Specify values for the following fields:
++
+* *Bucket name* +
+Name of the bucket containing the multipart upload to abort
+* *Notification queue name* +
+Amazon SQS queue name used to notify users when a new object is created or deleted in an S3 bucket.
+If omitted, a default queue name with prefix MULE-S3-TRIGGER-<event-type>-QUEUE-<'bucket-name'> is created and used. Where even-type is either NEW/REMOVED.
+NOTE: As the default queue is created (if not specified) and the source uses SQS:Received to poll for the event notifications connector needs following SQS permissions: sqs:CreateQueue, sqs: GetQueueAttributes, sqs: SetQueueAttributes, sqs: GetQueueUrl, sqs: ReceiveMessage, sqs: DeleteMessage*
```

Comment:
This is confusing and should be rewritten. I think it is missing some "the"s.
Also:
As > Because (as implies time)
SQS:Received > SQS *Received* operation??? (not sure if this is an operation) - global for the other SQS operations in this text.